types/daum-postcode/index.d.ts

```diff
@@ -0,0 +1,500 @@
+// Type definitions for non-npm package Daum Postcode 2.0
+// Project: https://postcode.map.daum.net/guide
+// Definitions by: doinki <https://github.com/doinki>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare global {
+    /** @see [Daum Postcode](https://postcode.map.daum.net/guide) */
+    const daum: Daum;
+
+    interface Window {
+        /** @see [Daum Postcode](https://postcode.map.daum.net/guide) */
+        readonly daum: Daum;
+    }
+}
```

Comment:
```suggestion
declare global {
    /** @see [Daum Postcode](https://postcode.map.daum.net/guide) */
    const daum: Daum;

    interface Window {
        /** @see [Daum Postcode](https://postcode.map.daum.net/guide) */
        readonly daum: Daum;
    }
}
```
this is not and UMD or CJS module, just global script, so it should be possible to use only this:
```ts
/** @see [Daum Postcode](https://postcode.map.daum.net/guide) */
declare var daum: Daum;
```
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-d-ts.html#global-libraries
(note that this would require `declare class`) and remove immport from tests

Reply:
thanks. I corrected it by referring to that link.